tools/ci/azure/pip_install.yml

```diff
@@ -0,0 +1,9 @@
+parameters:
+  packages: ''
+
+steps:
+- script: |
+    sudo easy_install pip
+    pip install --user ${{ parameters.packages }}
+    echo "##vso[task.prependpath]$HOME/Library/Python/2.7/bin"
```

Comment:
We are already modifying a bunch of system configs and installing packages to global. What about we just `sudo pip install --system` here? That will save us the trouble of modifying PATH.

Reply:
Unfortunately tox has a dependency which isn't possible to upgrade on the system because it's somehow protected. setuptools I think. I'll try to make the suggested change and then document why it doesn't work.

Reply:
It'll be System Integrity Protection of the system Python.

Reply:
Oh right... I brew Python on my mac so I forgot there's that...